portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -63,4 +69,14 @@ private String getParameter(HttpServletRequest request, String name, String defa
     }
     return value;
   }
+
+  /**
+   * Stores user comments in a form of persistent storage.
+   */
+  private void storeComments(String name, String comment) {
```

Comment:
Try to make the function static if possible.

Reply:
Done.